docs/diagrams/UiSequenceDiagram.puml

```diff
@@ -1,4 +1,6 @@
 @startuml
+!include docs/diagrams/Style.puml
```

Comment:
Does this path actually work? The UiSequencediagram.puml is already in docs/diagrams directory. Should it be just Style.puml?